From 4ecca5e1c02030fc086cc664c4410c3c7c59814d Mon Sep 17 00:00:00 2001
From: Jianzheng Zhou <jianzheng.zhou@freescale.com>
Date: Wed, 21 May 2014 10:30:03 +0800
Subject: [PATCH 39/39] ENGR00305958 ethernet:add support for multiple
 ethernet interface

If more than one ethernet interfaces exist, just also up it and monitor it.

Signed-off-by: Jianzheng Zhou <jianzheng.zhou@freescale.com>
---
 core/java/android/net/EthernetDataTracker.java | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/core/java/android/net/EthernetDataTracker.java b/core/java/android/net/EthernetDataTracker.java
index 5c88546..b389f48 100644
--- a/core/java/android/net/EthernetDataTracker.java
+++ b/core/java/android/net/EthernetDataTracker.java
@@ -1,6 +1,6 @@
 /*
  * Copyright (C) 2010 The Android Open Source Project
- * Copyright (C) 2013 Freescale Semiconductor, Inc.
+ * Copyright (C) 2013-2014 Freescale Semiconductor, Inc.
  *
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
@@ -253,7 +253,6 @@ public class EthernetDataTracker extends BaseNetworkStateTracker {
                     NetworkUtils.stopDhcp(mIface);
 
                     reconnect();
-                    break;
                 }
             }
         } catch (RemoteException e) {
-- 
1.8.0

